### ISSUE TITLE
On Windows 11 24H2 26100.XXX didn't detect CPU clocks - just  infinity symbol.

### ISSUE THREAD

**Issue by UnclWish** [openhardwaremonitor/openhardwaremonitor#1611]
Just updated to Windows 11 24H2 26100.560 and OHM start to show my CPU clocks on every core as infinity symbol so as bus clock.
MB - ASUS SaberTooth 990FX R2.0 CPU - AMD FX-8320.

**Comment by UnclWish** [comment]
Found a cause of this bug. Windows after update for some reason resets compatibility flag to run as administrator from Openhardwaremonitor.exe
Setting this flag again and running as admin fixes the issue.

**Comment by UnclWish** [comment]
Next Windows 11 24H2 update to build 26100.712 again broke bus and cores clocks reading... Running as admin didn't helps... What wrong with this 24H2?..

**Comment by Taiizor** [comment]
This project is no longer being developed. I recommend using this instead: https://github.com/LibreHardwareMonitor/LibreHardwareMonitor

**Comment by UnclWish** [comment]
> This project is no longer being developed. I recommend using this instead: https://github.com/LibreHardwareMonitor/LibreHardwareMonitor

I know it. But it's no reason for me to use it. Just some more sensors. And this bug is on it too. Something wrong with .sys file created every launch of program...

**Comment by Taiizor** [comment]
> > This project is no longer being developed. I recommend using this instead: https://github.com/LibreHardwareMonitor/LibreHardwareMonitor
> 
> I know it. But it's no reason for me to use it. Just some more sensors. And this bug is on it too. Something wrong with .sys file created every launch of program...

In that case, the root cause of the issue is WinRing0.

**Comment by UnclWish** [comment]
Yes, winring0 is the cause. After several launches of Libre and Open HM clocks are read. Can't understand what is going on. But it's somehow linked with 24H2. On 23H2 all works fine.

**Comment by Taiizor** [comment]
Actually, I am using 24H2 (26120.670), and the application works without any issues.

![LibreHardwareMonitor](https://github.com/openhardwaremonitor/openhardwaremonitor/assets/41683699/9425b65c-5423-4074-b264-b1045530ff78)

**Comment by UnclWish** [comment]
Your build is on other channel. 26120 is insider dev build. 26100 is stable release. I didn't know what's wrong...
